packages/avatar-group/src/vaadin-avatar-group.js

```diff
@@ -393,7 +393,7 @@ class AvatarGroup extends ResizeMixin(ElementMixin(ThemableMixin(ControllerMixin
 
   /** @private */
   _onListKeyDown(event) {
-    if (event.key === 'Escape' || event.key === 'Esc' || /^(Tab)$/.test(event.key)) {
+    if (event.key === 'Escape' || event.key === 'Esc' || /^(Tab)$/u.test(event.key)) {
```

Comment:
This could be simplified as follows:

```suggestion
    if (event.key === 'Escape' || event.key === 'Tab') {
```

We don't need `Esc` as it was used by IE11 and legacy Edge. Also we can use `Tab`.